submit-test-status/README.md

```diff
@@ -0,0 +1,83 @@
+
+# submit-test-status
+
+This composite Github Action (GHA) is aimed to be used by Camunda teams for submitting the test status(es) from a GHA workflow job to [CI Analytics](https://confluence.camunda.com/display/HAN/CI+Analytics). This allows teams to generate statistics about flakiness, runtime, etc. per test case across all branches and prioritize the worst offenders.
+
+## Usage
+
+This composite GHA can be used in any repository that was set up to provide credentials for accessing the central Google Big Query (BQ) database (credentials via Vault).
+
+### Inputs
+
+| Input name           | Description                                        |
+|----------------------|----------------------------------------------------|
+| test_event_record    | Multi-line string that contains the details of the test events in JSONL format. One test event per line. |
```

Comment:
I'd appreciate a link on "JSONL format" to e.g. https://jsonlines.org/ so people can look up easily what it stands for.